phalcon/mvc/model/query.zep

```diff
@@ -36,6 +38,8 @@ use Phalcon\Mvc\Model\ResultsetInterface;
 use Phalcon\Mvc\Model\Resultset\Simple;
 use Phalcon\Di\InjectionAwareInterface;
 use Phalcon\Mvc\Model\RelationInterface;
+use Phalcon\Mvc\Model\TransactionInterface;
+use Phalcon\Mvc\Model as Model;
```

Comment:
that's embarrassing to admin .... the type hint you suggested with the modelinterface ... well I tried it with the model class and the compiler complained, and I overlooked it in the cleanup ... thx